demo/demo_pdf.py

```diff
@@ -0,0 +1,88 @@
+from schematics import Model, types, models
+
+from export_util import normalize, writer, Exporter, utility
+
+
+@models.metaclass(utility.ExportableModelMeta)
```

Comment:
The PDF Writer should be generic export format. 
We don't need any model here, the models should be in your individual libraries. 

The idea is to support any format by providing the Template with the Object and Field